src/neo/Ledger/Blockchain.cs

```diff
@@ -258,8 +252,13 @@ private void AddUnverifiedBlockToCache(Block block)
             {
                 blocks = new LinkedList<Block>();
                 block_cache_unverified.Add(block.Index, blocks);
+                return;
```

Comment:
I think that the `block` need to be added to `blocks` being adding.
```cs
blocks = new LinkedList<Block>();
blocks.AddLast(block);
block_cache_unverified.Add(block.Index, blocks);
```

Reply:
@ShawnYun, did you take a look at this?

Reply:
@vncoelho Thanks, you are right. Or we can delete this `return`, then the `block` will add to `blocks`?

Reply:
Not sure, @ShawnYun, aheuahuea